File: PraktikumModul3/PRAK304_2110817210013_MuhammadChairurRijal.py
a = int(input(""));

if a > 0 & a < 10 :
    print("Satuan");
elif a > 10 & a < 20 :
    print("Belasan");
elif a == 0 :
    print("Nol");
elif a >= 100 :
    print("Anda Mengimput Meliputi Limit Bilangan");
else:
    print("Puluhan");   

File: PraktikumModul3/PRAK304_2110817210013_MuhammadChairurRijal.c
#include <stdio.h>

int main() {
    int a;
    scanf("%d", &a);
    if ( a > 0 && a < 10 ) {
        printf("Satuan");
    } else if ( a >= 10 && a <= 20){
        printf("Belasan");
    } else if ( a == 0 ) {
        printf("Nol");
    } else if ( a >= 100 ) {
        printf("Anda Mengimput Meliputi Limit Bilangan");
    } else {
        printf("Puluhan");
    }
}